Sources/_StringProcessing/ByteCodeGen.swift

```diff
@@ -326,6 +382,9 @@ fileprivate extension Compiler.ByteCodeGen {
 
     builder.buildSave(success)
     builder.buildSave(intercept)
+    if reverse {
+       builder.buildReverse(1)
```

Comment:
What's the logic behind emitting this reverse?

Reply:
All the `match` instructions match the current position, so my logic was in order to start matching in reverse, we needed to step back once and _then_ start the match

Reply:
Are we going to be emitting the reverse variant of those matching instructions?

Reply:
I believe so. Assuming the save point machinery doesn't have to change, we call `emitNode` for the pattern inside the lookbehind so it should be reversed so long as `reverse` is true.

Reply:
So long as the `reverse` flag is set, yes. Thinking about nested lookarounds though, I think my current implementation of the `reverse` flag will have a bug here because of what you mentioned in another comment

> Are nested lookarounds possible? If so, do we need to restore the prior reverse value instead of clearing it?